packages/dmn-js/CHANGELOG.md

```diff
@@ -6,6 +6,12 @@ All notable changes to [dmn-js](https://github.com/bpmn-io/dmn-js) are documente
 
 ___Note:__ Yet to be released changes appear here._
 
+## 9.2.0
+
+* `FEAT`: close select whenever there is user interaction outside of it ([#546](https://github.com/bpmn-io/dmn-js/issues/546), [#559](https://github.com/bpmn-io/dmn-js/issues/559))
```

Comment:
To make it easier to understand for the user maybe put: "close decision table hit-policy select"

Reply:
The feature changes behaviour of all selects in the library, so it's not only hit policy select.